src/conditional-compilation.md

```diff
@@ -196,17 +196,19 @@ Example values:
 * `"msvc"`
 * `"musl"`
 * `"sgx"`
+* `"sim"`
+* `"macabi"`
 
 r[cfg.target_abi]
 ### `target_abi`
 
 r[cfg.target_abi.general]
-Key-value option set to further disambiguate the `target_env` with information
-about the target ABI.
+Key-value option set to further disambiguate the `target_env` or `target_arch`
```

Comment:
I was wondering if we could make this more general about what it is disambiguating? (Instead of being specific about `env` or `arch`.) Maybe something like this?


```suggestion
Key-value option set to further disambiguate the target
```

This mirrors the wording in `target_env`. My reasoning is that we don't have any specific rules that the `target_abi` has to be related to any particular field.

Reply:
I'm fine with that, have done so in the latest commit.